scripts/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ "${UPDATE_ON_BOOT}" = false ]; then
+    echo "Update on Boot needs to be enabled for auto updating"
+    exit 1
+fi
+
+CURRENTBUILD=$(awk '/buildid/{ print $2 }' < /palworld/steamapps/appmanifest_2394010.acf)
+TARGETBUILD=$(curl https://api.steamcmd.net/v1/info/2394010 --silent | grep -P '"public": {"buildid": "\d+"' -o | sed -r 's/.*("[0-9]+")$/\1/')
+
+if [ "$CURRENTBUILD" != "$TARGETBUILD" ]; then
```

Comment:
In the event that `TARGET_BUILD` is empty you shouldn't update.
This will happen if the site is unavailable. You can test by changing to an invalid site.
```suggestion
if [ -n "$TARGETBUILD" ] && [ "$CURRENTBUILD" != "$TARGETBUILD" ]; then
```

Reply:
Yes that's a good point. Didn't think of this case.